commands/createfixpullrequests.go

```diff
@@ -30,6 +30,8 @@ type CreateFixPullRequestsCmd struct {
 	projectWorkingDir string
 	// The git client the command performs git operations with
 	gitManager *utils.GitManager
+	// Decide if to fix each vulnerability separately or to aggregate to one pull request at any given time
```

Comment:
// Decide if to fix each vulnerability separately or to aggregate to one pull request at any given time
-->
// Determines whether to open a pull request for each vulnerability fix or to aggregate all fixes into one pull request.